src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -360,6 +361,22 @@
     FLUSH_LOCAL_SCHEMA_CHANGES("cassandra.test.flush_local_schema_changes", "true"),
 
     TOMBSTONE_HISTOGRAM_TTL_ROUND_SECONDS("cassandra.streaminghistogram.roundseconds", "60"),
+
+    /**
+     * This property indicates with what Cassandra major version the storage format will be compatible with.
+     *
+     * The chosen storage compatiblity mode will determine the versions of the written sstables, commitlogs, hints,
+     * etc. Those storage elements will use the higher minor versions of the major version that corresponds to the
+     * Cassandra version we want to stay compatible with. For example, if we want to stay compatible with Cassandra 4.0
+     * or 4.1, the value of this property should be 4, and that will make us use 'nc' sstables.
```

Comment:
I forgot this when changing from int to enum:
```suggestion
     * or 4.1, the value of this property should be {@code CASSANDRA_4}, and that will make us use 'nc' sstables.
```